rootfs-builder/rootfs.sh

```diff
@@ -247,6 +245,39 @@ error_handler()
 	fi
 }
 
+detect_go_version()
+{
+	typeset -r yq=$(command -v yq || command -v ${GOPATH}/bin/yq)
+	[ -z "$yq" ] && die "'yq' application not found (needed to parsing minimum Go version required)"
+
+	typeset -r runtimeVersion="${AGENT_VERSION:-master}"
```

Comment:
Ok, if user species  no `AGENT_VERSION` or `master`, then I am using the `master` runtime.
Otherwise, I am getting the runtime version from `agent/VERSION` file.